apispec/openapi.json

```diff
@@ -4773,7 +4773,7 @@
             },
             "in": "query",
             "name": "limit",
-            "description": "Indicates the maximum number of records to return, the default value is 10080 which means that it returns all data acquiescently."
+            "description": "Indicates the maximum number of records to return. The default value is 10080, which means that it returns all data acquiescently."
```

Comment:
Again,  I suggest deleting ", which means that it returns all data acquiescently."